pkg/openebs/openebs.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
nit : OpenEBS ?

Reply:
What should we put here? Only replace the Kubernetes or something different should go her altogether?